### DISCUSSION TITLE
Incompatible with react-countup

### DISCUSSION THREAD

**Discussion by arkmech** [egoist/tsup#1049] [Q&A]
Using `tsup` for React component library. A component uses `react-countup`.

`react-countup` works well in regular Next.js app consuming library. But when using the component from tsup generated library that uses `react-countup` under the hood. I get this error.

```
Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object. bundling
```

Is there a tsup configuration option I am suppose to have on?

My configuration:
```ts
export default defineConfig((options: Options) => ({
  entry: {
    'Button/index': 'src/components/atoms/Button/Button.tsx',
    // other components...
  },
  splitting: false,
  outDir: 'dist',
  format: ['esm', 'cjs'],
  dts: true,
  minify: true,
  external: ['react', 'next'],
  ...options,
}));
```

**Comment by ocavue** [comment]
Please provide a minimal reproduction. Thanks!
